client/src/components/add-username-modal/add-username-modal.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../scss/variables';
+@import '../../scss/colors';
+@import '../../scss/mixins';
+@import '../../scss/atoms/button';
+
+@import '../comment/comment.scss';
+
+.username-modal {
+  @include z-level(3);
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  position: fixed;
+  right: $baseline;
+  bottom: $baseline;
+  width: 35vw;
```

Comment:
@tobbe-j has this been tested at mobile? consider using max-width, min-width and same with heights

Reply:
Should now work decently on narrower screens too